chrome/content/zotero/xpcom/zotero.js

```diff
@@ -507,6 +509,35 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 			}
 		}
 		
+		// Check for DB purge
+		var newAccountFile = dataDir.clone();
+		newAccountFile.append('new-account');
```

Comment:
(And showing a prompt on a manual click of the Unlink button is hardly code complexity.)